session_stats.go

```diff
@@ -30,6 +30,10 @@ func (session *Session) Count(bean ...interface{}) (int64, error) {
 		args = session.statement.RawParams
 	}
 
+	if len(session.statement.selectStr) > 0 {
+		sqlStr = "SELECT COUNT(*) FROM ("+sqlStr+") _TEMP_"
```

Comment:
Why we need a subquery?

Reply:
> Why we need a subquery?

For "GroupBy" statement. We need to know the total records of the query.